src/playwrightTest.ts

```diff
@@ -166,12 +166,12 @@ export class PlaywrightTest {
       stdio: ['pipe', 'pipe', 'pipe', 'pipe', 'pipe'],
       env: {
         ...process.env,
-        ...settingsEnv,
         ...this._reusedBrowser.browserServerEnv(false),
         ...this._reusedBrowser.logApiCallsEnv(),
         ...(await reporterServer.env()),
         // Don't debug tests when running them.
         NODE_OPTIONS: undefined,
+        ...settingsEnv,
```

Comment:
Let's move NODE_OPTIONS to be before ...settingsEnv instead. That way users are not accidentally override the other internal env variables we rely upon (lines 169-171 new code).